packages/browser-tests/package.json

```
@@ -4,16 +4,31 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "start": "run-s start:*",
-    "start:setup": "node ./src/setup-example.js",
-    "start:build": "node ./src/build-example.js",
-    "start:serve": "npx serve -p 8080 --no-port-switching ./example",
-    "test": "npx playwright test"
+    "start": "npm run start:server",
```

nit: can we group similar commands, please?

```suggestion
    "start": "npm run start:server",
    "start:serve": "cd ../headless-tests && npx serve -p 8080 --no-port-switching . -s",
    "start:server": "node ./dist/server.js",
```